implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/v2/DynatraceExporterV2Test.java

```diff
@@ -332,8 +332,8 @@ void toMeterLine() {
 
     @Test
     void gaugeWithInvalidNameShouldBeDropped() {
-        meterRegistry.gauge("~~~", 1.23);
-        Gauge gauge = meterRegistry.find("~~~").gauge();
+        meterRegistry.gauge("", 1.23);
+        Gauge gauge = meterRegistry.find("").gauge();
```

Comment:
These tests had to be adapted because with the new normalization/sanitization rules followed by the library, this will be replaced with a valid metric identifier. An empty string will still provoke an invalid one.